### DISCUSSION TITLE
Meeting 4: May 2nd, 14:00 UTC

### DISCUSSION THREAD

**Discussion by PeterKraus** [marda-alliance/metadata_extractors#16] [Announcements]
## 4th MaRDA Extractors WG meeting: ELN roundtable

**When**: [2nd of May, 14:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230502T16&p1=37)
**Where**: https://meet.jit.si/marda_extractors

**Agenda**:
- Introductions and summary since last meeting (~5 minutes)
- ELN Roundtable (30 minutes)
- Discussion (until 15:00 UTC, open-ended)

During this meeting, we’d like to focus particularly on the "downstream" consumers of the packages that we plan to create in the WG, such as ELN or data pipeline developers. I would like to come out of the meeting with a "wishlist" for features, ideally with assigned feature priorities, so that we can focus on delivering a useful piece of software before the end of the WG (currently Oct. 2023).

I have already contacted several of you for feedback and input on the above. I would like to spend the first 30 minutes of the meeting in a roundtable with contributions from you folks (say, up to 3 minutes each), and the second half of the meeting with a discussion of those suggestions and prioritisation. If you cannot attend, or would rather send your feedback in writing rather than speak, please email me or use this Discussion.

**Comment by ml-evs** [comment]
Minutes are available here: https://github.com/marda-alliance/metadata_extractors/blob/main/meetings/4-meeting-2023-05-02/minutes.md

The meeting was also recorded, unfortunately as a QHD screen capture, so I just have a 2 GB video sitting on disk. I guess if anyone explicitly wants to see it, let me know!
